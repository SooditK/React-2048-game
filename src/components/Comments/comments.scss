@import '../../assets/styles/vars';

.main {
  margin: auto;
  width: 60vw;
}

.comments {
  border-radius: 5px;
  margin: 3.85vw auto 0;
  padding: 1.54vw;

  .comments-title {
    border-bottom: 1px solid $border-color;
    font-size: 1.25rem;
    font-weight: 500;
    padding-bottom: 0.77vw;
    text-align: left;

    span {
      color: $light-black;
      text-decoration: line-through;
    }
  }

  .container {
    text-align: left;
  }
}

// gitalk css
// stylelint-disable
:global {
  @font-face {
    font-family: octicons-link;
    src: url('data:font/woff;charset=utf-8;base64,d09GRgABAAAAAAZwABAAAAAACFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEU0lHAAAGaAAAAAgAAAAIAAAAAUdTVUIAAAZcAAAACgAAAAoAAQAAT1MvMgAAAyQAAABJAAAAYFYEU3RjbWFwAAADcAAAAEUAAACAAJThvmN2dCAAAATkAAAABAAAAAQAAAAAZnBnbQAAA7gAAACyAAABCUM+8IhnYXNwAAAGTAAAABAAAAAQABoAI2dseWYAAAFsAAABPAAAAZwcEq9taGVhZAAAAsgAAAA0AAAANgh4a91oaGVhAAADCAAAABoAAAAkCA8DRGhtdHgAAAL8AAAADAAAAAwGAACfbG9jYQAAAsAAAAAIAAAACABiATBtYXhwAAACqAAAABgAAAAgAA8ASm5hbWUAAAToAAABQgAAAlXu73sOcG9zdAAABiwAAAAeAAAAME3QpOBwcmVwAAAEbAAAAHYAAAB/aFGpk3jaTY6xa8JAGMW/O62BDi0tJLYQincXEypYIiGJjSgHniQ6umTsUEyLm5BV6NDBP8Tpts6F0v+k/0an2i+itHDw3v2+9+DBKTzsJNnWJNTgHEy4BgG3EMI9DCEDOGEXzDADU5hBKMIgNPZqoD3SilVaXZCER3/I7AtxEJLtzzuZfI+VVkprxTlXShWKb3TBecG11rwoNlmmn1P2WYcJczl32etSpKnziC7lQyWe1smVPy/Lt7Kc+0vWY/gAgIIEqAN9we0pwKXreiMasxvabDQMM4riO+qxM2ogwDGOZTXxwxDiycQIcoYFBLj5K3EIaSctAq2kTYiw+ymhce7vwM9jSqO8JyVd5RH9gyTt2+J/yUmYlIR0s04n6+7Vm1ozezUeLEaUjhaDSuXHwVRgvLJn1tQ7xiuVv/ocTRF42mNgZGBgYGbwZOBiAAFGJBIMAAizAFoAAABiAGIAznjaY2BkYGAA4in8zwXi+W2+MjCzMIDApSwvXzC97Z4Ig8N/BxYGZgcgl52BCSQKAA3jCV8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==')
      format('woff');
  }

  @media (max-width: 479px) {
    .gt-container .gt-avatar {
      height: 2em;
      width: 2em;
    }
  }

  @media (max-width: 479px) {
    .gt-container .gt-avatar-github {
      height: 1.875em;
      width: 1.875em;
    }
  }

  @media (max-width: 479px) {
    .gt-container .gt-header-comment {
      margin-left: 0.875em;
    }
  }

  @media (max-width: 479px) {
    .gt-container .gt-header-controls-tip {
      display: none;
    }
  }

  @media (max-width: 479px) {
    .gt-container .gt-header-controls .gt-btn {
      float: none;
      width: 100%;
    }
  }

  @media (max-width: 479px) {
    .gt-container .gt-comment-content {
      margin-left: 0.875em;
      padding: 0.625em 0.75em;
    }
  }

  @keyframes gt-kf-rotate {
    0% {
      transform: rotate(0);
    }

    100% {
      transform: rotate(360deg);
    }
  }

  .markdown-body {
    color: #24292e;
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial,
      sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
    font-size: 16px;
    line-height: 1.5;
    line-height: 1.5;
    -ms-text-size-adjust: 100%;
    -webkit-text-size-adjust: 100%;
    word-wrap: break-word;
  }

  .markdown-body .pl-c {
    color: #6a737d;
  }

  .markdown-body .pl-c1,
  .markdown-body .pl-s .pl-v {
    color: #005cc5;
  }

  .markdown-body .pl-e,
  .markdown-body .pl-en {
    color: #6f42c1;
  }

  .markdown-body .pl-smi,
  .markdown-body .pl-s .pl-s1 {
    color: #24292e;
  }

  .markdown-body .pl-ent {
    color: #22863a;
  }

  .markdown-body .pl-k {
    color: #d73a49;
  }

  .markdown-body .pl-s,
  .markdown-body .pl-pds,
  .markdown-body .pl-s .pl-pse .pl-s1,
  .markdown-body .pl-sr,
  .markdown-body .pl-sr .pl-cce,
  .markdown-body .pl-sr .pl-sre,
  .markdown-body .pl-sr .pl-sra {
    color: #032f62;
  }

  .markdown-body .pl-v,
  .markdown-body .pl-smw {
    color: #e36209;
  }

  .markdown-body .pl-bu {
    color: #b31d28;
  }

  .markdown-body .pl-ii {
    background-color: #b31d28;
    color: #fafbfc;
  }

  .markdown-body .pl-c2 {
    background-color: #d73a49;
    color: #fafbfc;
  }

  .markdown-body .pl-c2::before {
    content: '^M';
  }

  .markdown-body .pl-sr .pl-cce {
    color: #22863a;
    font-weight: bold;
  }

  .markdown-body .pl-ml {
    color: #735c0f;
  }

  .markdown-body .pl-mh,
  .markdown-body .pl-mh .pl-en,
  .markdown-body .pl-ms {
    color: #005cc5;
    font-weight: bold;
  }

  .markdown-body .pl-mi {
    color: #24292e;
    font-style: italic;
  }

  .markdown-body .pl-mb {
    color: #24292e;
    font-weight: bold;
  }

  .markdown-body .pl-md {
    background-color: #ffeef0;
    color: #b31d28;
  }

  .markdown-body .pl-mi1 {
    background-color: #f0fff4;
    color: #22863a;
  }

  .markdown-body .pl-mc {
    background-color: #ffebda;
    color: #e36209;
  }

  .markdown-body .pl-mi2 {
    background-color: #005cc5;
    color: #f6f8fa;
  }

  .markdown-body .pl-mdr {
    color: #6f42c1;
    font-weight: bold;
  }

  .markdown-body .pl-ba {
    color: #586069;
  }

  .markdown-body .pl-sg {
    color: #959da5;
  }

  .markdown-body .pl-corl {
    color: #032f62;
    text-decoration: underline;
  }

  .markdown-body .octicon {
    display: inline-block;
    fill: currentColor;
    vertical-align: text-top;
  }

  .markdown-body a {
    background-color: transparent;
    -webkit-text-decoration-skip: objects;
  }

  .markdown-body a:active,
  .markdown-body a:hover {
    outline-width: 0;
  }

  .markdown-body strong {
    font-weight: inherit;
  }

  .markdown-body strong {
    font-weight: bolder;
  }

  .markdown-body h1 {
    font-size: 2em;
    margin: 0.67em 0;
  }

  .markdown-body img {
    border-style: none;
  }

  .markdown-body svg:not(:root) {
    overflow: hidden;
  }

  .markdown-body code,
  .markdown-body kbd,
  .markdown-body pre {
    font-family: monospace, monospace;
    font-size: 1em;
  }

  .markdown-body hr {
    box-sizing: content-box;
    height: 0;
    overflow: visible;
  }

  .markdown-body input {
    font: inherit;
    margin: 0;
  }

  .markdown-body input {
    overflow: visible;
  }

  .markdown-body [type='checkbox'] {
    box-sizing: border-box;
    padding: 0;
  }

  .markdown-body * {
    box-sizing: border-box;
  }

  .markdown-body input {
    font-family: inherit;
    font-size: inherit;
    line-height: inherit;
  }

  .markdown-body a {
    color: #0366d6;
    text-decoration: none;
  }

  .markdown-body a:hover {
    text-decoration: underline;
  }

  .markdown-body strong {
    font-weight: 600;
  }

  .markdown-body hr {
    background: transparent;
    border: 0;
    border-bottom: 1px solid #dfe2e5;
    height: 0;
    margin: 15px 0;
    overflow: hidden;
  }

  .markdown-body hr::before {
    content: '';
    display: table;
  }

  .markdown-body hr::after {
    clear: both;
    content: '';
    display: table;
  }

  .markdown-body table {
    border-collapse: collapse;
    border-spacing: 0;
  }

  .markdown-body td,
  .markdown-body th {
    padding: 0;
  }

  .markdown-body h1,
  .markdown-body h2,
  .markdown-body h3,
  .markdown-body h4,
  .markdown-body h5,
  .markdown-body h6 {
    margin-bottom: 0;
    margin-top: 0;
  }

  .markdown-body h1 {
    font-size: 32px;
    font-weight: 600;
  }

  .markdown-body h2 {
    font-size: 24px;
    font-weight: 600;
  }

  .markdown-body h3 {
    font-size: 20px;
    font-weight: 600;
  }

  .markdown-body h4 {
    font-size: 16px;
    font-weight: 600;
  }

  .markdown-body h5 {
    font-size: 14px;
    font-weight: 600;
  }

  .markdown-body h6 {
    font-size: 12px;
    font-weight: 600;
  }

  .markdown-body p {
    margin-bottom: 10px;
    margin-top: 0;
  }

  .markdown-body blockquote {
    margin: 0;
  }

  .markdown-body ul,
  .markdown-body ol {
    margin-bottom: 0;
    margin-top: 0;
    padding-left: 0;
  }

  .markdown-body ol ol,
  .markdown-body ul ol {
    list-style-type: lower-roman;
  }

  .markdown-body ul ul ol,
  .markdown-body ul ol ol,
  .markdown-body ol ul ol,
  .markdown-body ol ol ol {
    list-style-type: lower-alpha;
  }

  .markdown-body dd {
    margin-left: 0;
  }

  .markdown-body code {
    font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier,
      monospace;
    font-size: 12px;
  }

  .markdown-body pre {
    font: 12px 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier,
      monospace;
    margin-bottom: 0;
    margin-top: 0;
  }

  .markdown-body .octicon {
    vertical-align: text-bottom;
  }

  .markdown-body .pl-0 {
    padding-left: 0 !important;
  }

  .markdown-body .pl-1 {
    padding-left: 4px !important;
  }

  .markdown-body .pl-2 {
    padding-left: 8px !important;
  }

  .markdown-body .pl-3 {
    padding-left: 16px !important;
  }

  .markdown-body .pl-4 {
    padding-left: 24px !important;
  }

  .markdown-body .pl-5 {
    padding-left: 32px !important;
  }

  .markdown-body .pl-6 {
    padding-left: 40px !important;
  }

  .markdown-body::before {
    content: '';
    display: table;
  }

  .markdown-body::after {
    clear: both;
    content: '';
    display: table;
  }

  .markdown-body > *:first-child {
    margin-top: 0 !important;
  }

  .markdown-body > *:last-child {
    margin-bottom: 0 !important;
  }

  .markdown-body a:not([href]) {
    color: inherit;
    text-decoration: none;
  }

  .markdown-body .anchor {
    float: left;
    line-height: 1;
    margin-left: -20px;
    padding-right: 4px;
  }

  .markdown-body .anchor:focus {
    outline: none;
  }

  .markdown-body p,
  .markdown-body blockquote,
  .markdown-body ul,
  .markdown-body ol,
  .markdown-body dl,
  .markdown-body table,
  .markdown-body pre {
    margin-bottom: 16px;
    margin-top: 0;
  }

  .markdown-body hr {
    background-color: #e1e4e8;
    border: 0;
    height: 0.25em;
    margin: 24px 0;
    padding: 0;
  }

  .markdown-body blockquote {
    border-left: 0.25em solid #dfe2e5;
    color: #6a737d;
    padding: 0 1em;
  }

  .markdown-body blockquote > :first-child {
    margin-top: 0;
  }

  .markdown-body blockquote > :last-child {
    margin-bottom: 0;
  }

  .markdown-body kbd {
    background-color: #fafbfc;
    border: solid 1px #c6cbd1;
    border-bottom-color: #959da5;
    border-radius: 3px;
    box-shadow: inset 0 -1px 0 #959da5;
    color: #444d56;
    display: inline-block;
    font-size: 11px;
    line-height: 10px;
    padding: 3px 5px;
    vertical-align: middle;
  }

  .markdown-body h1,
  .markdown-body h2,
  .markdown-body h3,
  .markdown-body h4,
  .markdown-body h5,
  .markdown-body h6 {
    font-weight: 600;
    line-height: 1.25;
    margin-bottom: 16px;
    margin-top: 24px;
  }

  .markdown-body h1 .octicon-link,
  .markdown-body h2 .octicon-link,
  .markdown-body h3 .octicon-link,
  .markdown-body h4 .octicon-link,
  .markdown-body h5 .octicon-link,
  .markdown-body h6 .octicon-link {
    color: #1b1f23;
    vertical-align: middle;
    visibility: hidden;
  }

  .markdown-body h1:hover .anchor,
  .markdown-body h2:hover .anchor,
  .markdown-body h3:hover .anchor,
  .markdown-body h4:hover .anchor,
  .markdown-body h5:hover .anchor,
  .markdown-body h6:hover .anchor {
    text-decoration: none;
  }

  .markdown-body h1:hover .anchor .octicon-link,
  .markdown-body h2:hover .anchor .octicon-link,
  .markdown-body h3:hover .anchor .octicon-link,
  .markdown-body h4:hover .anchor .octicon-link,
  .markdown-body h5:hover .anchor .octicon-link,
  .markdown-body h6:hover .anchor .octicon-link {
    visibility: visible;
  }

  .markdown-body h1 {
    border-bottom: 1px solid #eaecef;
    font-size: 2em;
    padding-bottom: 0.3em;
  }

  .markdown-body h2 {
    border-bottom: 1px solid #eaecef;
    font-size: 1.5em;
    padding-bottom: 0.3em;
  }

  .markdown-body h3 {
    font-size: 1.25em;
  }

  .markdown-body h4 {
    font-size: 1em;
  }

  .markdown-body h5 {
    font-size: 0.875em;
  }

  .markdown-body h6 {
    color: #6a737d;
    font-size: 0.85em;
  }

  .markdown-body ul,
  .markdown-body ol {
    padding-left: 2em;
  }

  .markdown-body ul ul,
  .markdown-body ul ol,
  .markdown-body ol ol,
  .markdown-body ol ul {
    margin-bottom: 0;
    margin-top: 0;
  }

  .markdown-body li > p {
    margin-top: 16px;
  }

  .markdown-body li + li {
    margin-top: 0.25em;
  }

  .markdown-body dl {
    padding: 0;
  }

  .markdown-body dl dt {
    font-size: 1em;
    font-style: italic;
    font-weight: 600;
    margin-top: 16px;
    padding: 0;
  }

  .markdown-body dl dd {
    margin-bottom: 16px;
    padding: 0 16px;
  }

  .markdown-body table {
    display: block;
    overflow: auto;
    width: 100%;
  }

  .markdown-body table th {
    font-weight: 600;
  }

  .markdown-body table th,
  .markdown-body table td {
    border: 1px solid #dfe2e5;
    padding: 6px 13px;
  }

  .markdown-body table tr {
    background-color: #fff;
    border-top: 1px solid #c6cbd1;
  }

  .markdown-body table tr:nth-child(2n) {
    background-color: #f6f8fa;
  }

  .markdown-body img {
    background-color: #fff;
    box-sizing: content-box;
    max-width: 100%;
  }

  .markdown-body code {
    background-color: rgba(27, 31, 35, 0.05);
    border-radius: 3px;
    font-size: 85%;
    margin: 0;
    padding: 0;
    padding-bottom: 0.2em;
    padding-top: 0.2em;
  }

  .markdown-body code::before,
  .markdown-body code::after {
    content: '\A0';
    letter-spacing: -0.2em;
  }

  .markdown-body pre {
    word-wrap: normal;
  }

  .markdown-body pre > code {
    background: transparent;
    border: 0;
    font-size: 100%;
    margin: 0;
    padding: 0;
    white-space: pre;
    word-break: normal;
  }

  .markdown-body .highlight {
    margin-bottom: 16px;
  }

  .markdown-body .highlight pre {
    margin-bottom: 0;
    word-break: normal;
  }

  .markdown-body .highlight pre,
  .markdown-body pre {
    background-color: #f6f8fa;
    border-radius: 3px;
    font-size: 85%;
    line-height: 1.45;
    overflow: auto;
    padding: 16px;
  }

  .markdown-body pre code {
    background-color: transparent;
    border: 0;
    display: inline;
    line-height: inherit;
    margin: 0;
    max-width: auto;
    overflow: visible;
    padding: 0;
    word-wrap: normal;
  }

  .markdown-body pre code::before,
  .markdown-body pre code::after {
    content: normal;
  }

  .markdown-body .full-commit .btn-outline:not(:disabled):hover {
    border-color: #005cc5;
    color: #005cc5;
  }

  .markdown-body kbd {
    background-color: #fafbfc;
    border: solid 1px #d1d5da;
    border-bottom-color: #c6cbd1;
    border-radius: 3px;
    box-shadow: inset 0 -1px 0 #c6cbd1;
    color: #444d56;
    display: inline-block;
    font: 11px 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier,
      monospace;
    line-height: 10px;
    padding: 3px 5px;
    vertical-align: middle;
  }

  .markdown-body :checked + .radio-label {
    border-color: #0366d6;
    position: relative;
    z-index: 1;
  }

  .markdown-body .task-list-item {
    list-style-type: none;
  }

  .markdown-body .task-list-item + .task-list-item {
    margin-top: 3px;
  }

  .markdown-body .task-list-item input {
    margin: 0 0.2em 0.25em -1.6em;
    vertical-align: middle;
  }

  .markdown-body hr {
    border-bottom-color: #eee;
  }
  /* variables */
  /* functions & mixins */
  /* variables - calculated */
  /* styles */
  .gt-container {
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
    font-size: 16px;
    /* loader */
    /* error */
    /* initing */
    /* no int */
    /* link */
    /* meta */
    /* popup */
    /* header */
    /* comments */
    /* comment */
  }

  .gt-container * {
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
  }

  .gt-container a {
    color: #6190e8;
  }

  .gt-container a:hover {
    border-color: #81a6ed;
    color: #81a6ed;
  }

  .gt-container a.is--active {
    color: #333;
    cursor: default !important;
  }

  .gt-container a.is--active:hover {
    color: #333;
  }

  .gt-container .gt-svg {
    display: inline-block;
    height: 1em;
    vertical-align: sub;
    width: 1em;
  }

  .gt-container .gt-svg svg {
    fill: #6190e8;
    width: 100%;
  }

  .gt-container .gt-ico {
    display: inline-block;
  }

  .gt-container .gt-ico-text {
    margin-left: 0.3125em;
  }

  .gt-container .gt-ico-github .gt-svg {
    height: 100%;
    width: 100%;
  }

  .gt-container .gt-ico-github svg {
    fill: inherit;
  }

  .gt-container .gt-spinner {
    position: relative;
  }

  .gt-container .gt-spinner::before {
    animation: gt-kf-rotate 0.6s linear infinite;
    border: 1px solid #fff;
    border-radius: 50%;
    border-top-color: #6190e8;
    box-sizing: border-box;
    content: '';
    height: 0.75em;
    margin-left: -0.375em;
    margin-top: -0.1875em;
    position: absolute;
    top: 3px;
    width: 0.75em;
  }

  .gt-container .gt-loader {
    animation: ease gt-kf-rotate 1.5s infinite;
    border: 1px solid #999;
    border-radius: 50%;
    display: inline-block;
    font-style: normal;
    height: 1.75em;
    line-height: 1.75em;
    position: relative;
    width: 1.75em;
  }

  .gt-container .gt-loader:before {
    background-color: #999;
    border-radius: 50%;
    content: '';
    display: block;
    height: 0.375em;
    left: 50%;
    margin-left: -0.1875em;
    margin-top: -0.1875em;
    position: absolute;
    top: 0;
    width: 0.375em;
  }

  .gt-container .gt-avatar {
    display: inline-block;
    height: 3.125em;
    width: 3.125em;
  }

  .gt-container .gt-avatar img {
    border-radius: 3px;
    height: auto;
    width: 100%;
  }

  .gt-container .gt-avatar-github {
    height: 3em;
    width: 3em;
  }

  .gt-container .gt-btn {
    background-color: #6190e8;
    border: none;
    border-radius: 5px;
    color: #fff;
    cursor: pointer;
    display: inline-block;
    font-size: 0.75em;
    line-height: 1;
    outline: none;
    padding: 0.75em 1em;
    text-decoration: none;
    white-space: nowrap;
  }

  .gt-container .gt-btn:hover {
    background-color: #81a6ed;
    color: #fff;
  }

  .gt-container .gt-btn-text {
    font-weight: 400;
  }

  .gt-container .gt-btn-loading {
    display: inline-block;
    height: 1em;
    margin-left: 0.5em;
    position: relative;
    vertical-align: top;
    width: 0.75em;
  }

  .gt-container .gt-btn.is--disable {
    cursor: not-allowed;
    opacity: 0.5;
  }

  .gt-container .gt-btn-login {
    margin-right: 0;
  }

  .gt-container .gt-error {
    color: #ff3860;
    margin: 0.625em;
    text-align: center;
  }

  .gt-container .gt-initing {
    padding: 1.25em 0;
    text-align: center;
  }

  .gt-container .gt-initing-text {
    font-size: 92%;
    margin: 0.625em auto;
  }

  .gt-container .gt-no-init {
    padding: 1.25em 0;
    text-align: center;
  }

  .gt-container .gt-link {
    border-bottom: 1px dotted #6190e8;
  }

  .gt-container .gt-link-counts,
  .gt-container .gt-link-project {
    text-decoration: none;
  }

  .gt-container .gt-meta {
    border-bottom: 1px solid #e9e9e9;
    font-size: 1em;
    margin: 1.25em 0;
    padding: 1em 0;
    position: relative;
    position: relative;
    z-index: 10;
  }

  .gt-container .gt-meta:before,
  .gt-container .gt-meta:after {
    content: ' ';
    display: table;
  }

  .gt-container .gt-meta:after {
    clear: both;
  }

  .gt-container .gt-counts {
    margin: 0 0.625em 0 0;
  }

  .gt-container .gt-user {
    float: right;
    font-size: 92%;
    margin: 0;
  }

  .gt-container .gt-user-pic {
    height: 16px;
    margin-right: 0.5em;
    vertical-align: top;
    width: 16px;
  }

  .gt-container .gt-user-inner {
    cursor: pointer;
    display: inline-block;
  }

  .gt-container .gt-user .gt-ico {
    margin: 0 0 0 0.3125em;
  }

  .gt-container .gt-user .gt-ico svg {
    fill: inherit;
  }

  .gt-container .gt-user .is--poping .gt-ico svg {
    fill: #6190e8;
  }

  .gt-container .gt-version {
    color: #a1a1a1;
    margin-left: 0.375em;
  }

  .gt-container .gt-copyright {
    border-top: 1px solid #e9e9e9;
    margin: 0 0.9375em 0.5em;
    padding-top: 0.5em;
  }

  .gt-container .gt-popup {
    background: #fff;
    border: 1px solid #e9e9e9;
    display: inline-block;
    font-size: 0.875em;
    letter-spacing: 0.5px;
    padding: 0.625em 0;
    position: absolute;
    right: 0;
    top: 2.375em;
  }

  .gt-container .gt-popup .gt-action {
    cursor: pointer;
    display: block;
    margin: 0.5em 0;
    padding: 0 1.125em;
    position: relative;
    text-decoration: none;
  }

  .gt-container .gt-popup .gt-action.is--active:before {
    background: #6190e8;
    content: '';
    height: 0.25em;
    left: 0.5em;
    position: absolute;
    top: 0.4375em;
    width: 0.25em;
  }

  .gt-container .gt-header {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    position: relative;
  }

  .gt-container .gt-header-comment {
    -webkit-box-flex: 1;
    -ms-flex: 1;
    flex: 1;
    margin-left: 1.25em;
  }

  .gt-container .gt-header-textarea {
    background-color: #f6f6f6;
    border: 1px solid rgba(0, 0, 0, 0.1);
    border-radius: 5px;
    box-sizing: border-box;
    display: block;
    font-size: 0.875em;
    max-height: 15em;
    min-height: 5.125em;
    outline: none;
    padding: 0.75em;
    resize: vertical;
    transition: all 0.25s ease;
    width: 100%;
    word-wrap: break-word;
  }

  .gt-container .gt-header-textarea:hover {
    background-color: #fbfbfb;
  }

  .gt-container .gt-header-controls {
    margin: 0.75em 0 0;
    position: relative;
  }

  .gt-container .gt-header-controls:before,
  .gt-container .gt-header-controls:after {
    content: ' ';
    display: table;
  }

  .gt-container .gt-header-controls:after {
    clear: both;
  }

  .gt-container .gt-header-controls-tip {
    color: #6190e8;
    font-size: 0.875em;
    text-decoration: none;
    vertical-align: sub;
  }

  .gt-container .gt-header-controls .gt-btn {
    float: right;
  }

  .gt-container:after {
    bottom: 100%;
    content: '';
    left: 0;
    opacity: 0;
    position: fixed;
    right: 0;
    top: 0;
  }

  .gt-container.gt-input-focused {
    position: relative;
  }

  .gt-container.gt-input-focused:after {
    background: #000;
    bottom: 0%;
    content: '';
    left: 0;
    opacity: 0.6;
    position: fixed;
    right: 0;
    top: 0;
    transition: opacity 0.3s, bottom 0s;
    z-index: 9999;
  }

  .gt-container.gt-input-focused .gt-header-comment {
    z-index: 10000;
  }

  .gt-container .gt-comments {
    padding-top: 1.25em;
  }

  .gt-container .gt-comments-null {
    text-align: center;
  }

  .gt-container .gt-comments-controls {
    margin: 1.25em 0;
    text-align: center;
  }

  .gt-container .gt-comment {
    display: flex;
    padding: 0.625em 0;
    position: relative;
  }

  .gt-container .gt-comment-content {
    background-color: #f9f9f9;
    flex: 1;
    margin-left: 1.25em;
    overflow: auto;
    padding: 0.75em 1em;
    transition: all ease 0.25s;
  }

  .gt-container .gt-comment-content:hover {
    -webkit-box-shadow: 0 0.625em 3.75em 0 #f4f4f4;
    box-shadow: 0 0.625em 3.75em 0 #f4f4f4;
  }

  .gt-container .gt-comment-header {
    font-size: 0.875em;
    margin-bottom: 0.5em;
    position: relative;
  }

  .gt-container .gt-comment-username {
    color: #6190e8;
    font-weight: 500;
    text-decoration: none;
  }

  .gt-container .gt-comment-username:hover {
    text-decoration: underline;
  }

  .gt-container .gt-comment-text {
    color: #a1a1a1;
    margin-left: 0.5em;
  }

  .gt-container .gt-comment-date {
    color: #a1a1a1;
    margin-left: 0.5em;
  }

  .gt-container .gt-comment-edit,
  .gt-container .gt-comment-reply {
    height: 1.375em;
    position: absolute;
    right: 0;
    top: 0;
    width: 1.375em;
  }

  .gt-container .gt-comment-edit:hover,
  .gt-container .gt-comment-reply:hover {
    cursor: pointer;
  }

  .gt-container .gt-comment-body {
    color: #333 !important;
  }

  .gt-container .gt-comment-admin .gt-comment-content {
    background-color: #f6f9fe;
  }
}
