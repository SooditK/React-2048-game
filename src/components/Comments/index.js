import React, { Component } from 'react';
import gitalk from '../../utils/gitalk';
import styles from './comments.scss';
import i18n from '../../utils/i18n';

export default class Comments extends Component {
  componentDidMount() {
    if (process.env.NODE_ENV !== 'test') {
      gitalk.render('gitalk-container');
    }
  }

  render() {
    return (
      <div className={styles.main}>
        <div className={styles.comments}>
          {/* eslint-disable react/no-danger */}
          <p
            className={styles.commentsTitle}
            dangerouslySetInnerHTML={{ __html: i18n.commentTitle }}
          />
          <div id="gitalk-container" className={styles.container} />
        </div>
      </div>
    );
  }
}
