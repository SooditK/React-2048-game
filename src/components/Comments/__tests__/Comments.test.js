import React from 'react';
import renderer from 'react-test-renderer';
import Comments from '..';

describe('<Comments />', () => {
  it('component render', () => {
    const comments = renderer.create(<Comments />).toJSON();
    expect(comments).toMatchSnapshot();
  });
});
