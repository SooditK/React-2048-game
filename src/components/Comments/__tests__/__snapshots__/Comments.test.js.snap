// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Comments /> component render 1`] = `
<div
  className="main"
>
  <div
    className="comments"
  >
    <p
      className="commentsTitle"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Welcome to leave comments",
        }
      }
    />
    <div
      className="container"
      id="gitalk-container"
    />
  </div>
</div>
`;
